### PR TITLE
Implement component Logo 

### PR DESCRIPTION
**Pull request by thanhtrantruong47**
Implement component Logo 

image

### CODE REVIEW

File: examples/practice-typescript/src/components/Logo/Logo.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import './logo.scss';
+
+const Logo = () => {
+  return (
+    <div className="logo-group">
+      <a className="icon-logo" href="javascript:void(0)"></a>
```
Comment (hungdo-agilityio) [inline examples/practice-typescript/src/components/Logo/Logo.tsx]: why href is `javascript:void(0)`

File: examples/practice-typescript/src/components/Logo/Logo.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+      <h1 className="title">Company name</h1>
```
Comment (hungdo-agilityio) [inline examples/practice-typescript/src/components/Logo/Logo.tsx]: So do we have any behavior on this Company name? usually on logo we can click any where to navigate back to the homepage

File: examples/practice-typescript/src/components/Logo/logo.scss [hunk 3]
```diff
@@ -0,0 +1,18 @@
+.icon-logo {
+  background-image: url('../../assets/images/managers/logo-manager.svg');
```
Comment (hungdo-agilityio) [inline examples/practice-typescript/src/components/Logo/logo.scss]: Why use background image? We can use svg component. And also svg should including the company name not separate
https://blog.logrocket.com/how-to-use-svgs-react/

Comment (thanhtrantruong47) [reply examples/practice-typescript/src/components/Logo/logo.scss]: Fixed

File: examples/practice-typescript/src/components/Logo/Logo.tsx (hunk 2 above)
Comment (thanhtrantruong47) [reply examples/practice-typescript/src/components/Logo/Logo.tsx]: Fixed

File: examples/practice-typescript/src/components/Logo/Logo.tsx (hunk 1 above)
Comment (thanhtrantruong47) [reply examples/practice-typescript/src/components/Logo/Logo.tsx]: Fixed

**Review by thanhnguyen-agilityio — APPROVED** [review]

**Pull request merged**